src/ServiceControl.Config/Framework/RaygunFeedBack.cs

```diff
@@ -1,44 +1,58 @@
-﻿namespace ServiceControl.Config.Framework
+﻿#nullable enable
+
+namespace ServiceControl.Config.Framework
 {
     using System;
     using System.IO;
+    using System.Net;
+    using System.Net.Http;
+    using System.Reflection;
     using System.Threading.Tasks;
     using Extensions;
     using Mindscape.Raygun4Net;
 
-    public class RaygunFeedback : RaygunReporter
+    public sealed class RaygunFeedback : IRaygunUserProvider
     {
         public RaygunFeedback()
         {
-            InitializeTrackingId();
+            trackingId = GetOrSetTrackingId();
+
+            raygunClient = new RaygunClient(raygunSettings, this);
+
+            init = Task.Run(async () =>
+            {
+                enabled = await TryInitializeRaygunClientWithCredentials() ||
+                          await TryInitializeRaygunClientWithCredentials(CredentialCache.DefaultCredentials) ||
+                          await TryInitializeRaygunClientWithCredentials(CredentialCache.DefaultNetworkCredentials);
+                version = GetVersion();
```

Comment:
I was assuming someone put it into the task run not to slow down the start, which I find quite silly since it is a very simple reflection, but I figured I'll leave it here for now.